packages/panzer/disc-fe/src/Panzer_IntrepidOrientation.cpp

```diff
@@ -176,7 +176,7 @@ namespace panzer {
           vertices(connMgr.getConnectivity(lid),numVerticesPerCell);
 
         // This function call expects a view for the vertices, not the nodes
-        orts[lid] = Intrepid2::Orientation::getOrientation(cellTopo, vertices);
```

Comment:
Huh. How did this work before?

Reply:
Because of how our test suite built the inline mesh for unit testing. Going to follow this up with some more changes to get a test to cover the corner case.